### DISCUSSION TITLE
Writing SPSS files in chunks

### DISCUSSION THREAD

**Discussion by mnizol** [Roche/pyreadstat#142] [Q&A] [answered]
Is there any mechanism in pyreadstat to write SPSS files in chunks (or, alternatively, to append to an existing SPSS file)?  I'm running into OOM errors when writing large SPSS files.  Empirically, it appears that pyreadstat requires several times more memory to write an SPSS file than the size of the underlying file.

**Comment by ofajardo** [comment, marked as answer]
Unfortunately it is not possible. The underlying C library [Readstat](https://github.com/WizardMac/ReadStat) does not allow neither writing in chunks or append to a file
